docs/_layouts/tutorial_frame.html

```diff
@@ -19,8 +19,8 @@
 			margin: 0;
 		}
 		#map {
-			width: 600px;
```

Comment:
Should we make examples fullscreen?
@mourner 
@jonkoops

Reply:
I think we have no disadvantages with fullscreen. Additionally then there are no problem with mobile devices but this can be done with `@media` .

I personally don't like `height: 100vh`. I use in my projects `height: 100%` and add it to `html, body` too, but I don't know what is better